Release 34

```diff
@@ -0,0 +1,26 @@
+# Release -API Management service: April 2023
+A regular Azure API Management service updated was started February 9, 2023. This release concluded roll out March 31, 2023.
+
+### New features, fixes, and improvements
+
+1. Introduce new `context.Deployment.Gateway.*` context properties ([docs](https://learn.microsoft.com/en-us/azure/api-management/api-management-policy-expressions#ContextVariables))
+2. We fixed an issue, where the definition names in the API schema would display blank spaces.
+2. We fixed an issue, where the definition names in the API schema would display blank spaces.
+2. We fixed an issue, where the developer portal was hiding default domain, a parameter has been added to include default domain in the hostnames.
+3. We fixed an issue, where the API revisions were not being properly filtered.
+4. We implemented a fix that will allow users to remove built-in groups, previously where able to add built-in groups but could not remove them.
+5. We fixed an issue, where an error was received when creating API schemas with the same namespace.
+6. We fixed an issue, where policies contained all named values which caused a problem for secrets. The policy failed to check if given user had access to secret.
+7. We fixed an issue, whereby Azure Active Directory v1 tokens could be validated with `validate-azure-ad-token` policy, but v2 tokens could not be validated.  With this release, both v1 and v2 tokens are supported.
+8. Previously, validate-azure-ad-token would only validate tokens produced by the public cloud instance.  With this release, tokens produced in every Azure cloud can be validated.
```

Comment:
The validation is based on Azure AD and not specifically how the token was produced. 

Does this help clarify that? 
```suggestion
8. Previously, validate-azure-ad-token would only validate tokens against the public cloud instance of Azure Active Directory (AD).  With this release, token validation is supported in every Azure cloud where Azure Active Directory is available.
```